.github/workflows/CI.yml

```diff
@@ -127,6 +131,50 @@ jobs:
       - name: Create msi file using candle and light
         run: .\tools\packaging\windows\create_msi.ps1
 
+      - name: Install AWS Cli v2
+        run: |
+          Invoke-WebRequest -Uri "https://awscli.amazonaws.com/AWSCLIV2.msi" -OutFile "AWSCLIV2.msi"
+          msiexec.exe /i AWSCLIV2.msi /passive
+          [System.Environment]::SetEnvironmentVariable('Path',$Env:Path + ";C:\\Program Files\\Amazon\\AWSCLIV2",'User')
+
+      - name: Sign windows artifacts
+        run: |
+          $pkgfile = "build\packages\windows\amd64\aws-otel-collector.msi"
```

Comment:
would it be better if we have a separate file for this script? under https://github.com/aws-observability/aws-otel-collector/tree/main/tools/packaging

Reply:
That's a good idea, I'll refactor it out after verifying the package signing is working.